askbot/views/users.py

```diff
@@ -1487,12 +1487,45 @@ def user(request, id, tab_name=None):
         user_logged_in=profile_owner.is_authenticated,
     )
 
+    question_filter = {}
+    if request.user != user:
+        question_filter['is_anonymous'] = False
+
+    if askbot_settings.CONTENT_MODERATION_MODE == 'premoderation':
+        question_filter['approved'] = True
+
+    #
+    # Questions
+    #
+    questions_qs = profile_owner.posts.get_questions(
+                    user=request.user
+                ).filter(
+                    **question_filter
+                ).order_by(
+                    '-points'#, '-thread__last_activity_at' to match sorting with ajax loads
+                ).select_related(
+                    'thread', 'thread__last_activity_by'
+                )
```

Comment:
This style of indenting is consistent with the files we're editing, and probably isn't the way we'd indent it in Rover web.